detection_rules/etc/version.lock.json

```diff
@@ -7915,25 +7929,25 @@
       }
     },
     "rule_name": "Local Scheduled Task Creation",
-    "sha256": "da1e77e9fd2e8d8ee1411d69f4643ea5b67d88977558f3353a068201ef22fc3f",
+    "sha256": "1373b59fea16f6e151bcd9f897e78aae6f1df077b5e1d54bfdf95c93e26804dd",
     "type": "eql",
-    "version": 101
+    "version": 102
   },
   "b0046934-486e-462f-9487-0d4cf9e429c6": {
     "min_stack_version": "8.3",
     "previous": {
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "Timestomping using Touch Command",
-        "sha256": "4c674f73145d658fd82a2d0f0efbb8ab0da6d6e4158ab4cb6f38d490f02dd8c3",
+        "sha256": "03332052d7bcda03a20798da8475f4f192d2d0f46af22fd17630ff8952aab524",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.